src/CMakeLists.txt

```diff
@@ -161,7 +178,7 @@ set(ARCHIVE_ENABLE_CNG OFF CACHE BOOL "")
 set(ARCHIVE_ENABLE_MBEDTLS OFF CACHE BOOL "")
 set(ARCHIVE_ENABLE_NETTLE OFF CACHE BOOL "")
 set(ARCHIVE_ENABLE_OPENSSL OFF CACHE BOOL "")
-set(ARCHIVE_ENABLE_BZip2 OFF CACHE BOOL "")
+set(ARCHIVE_ENABLE_BZip2 ON CACHE BOOL "")
```

Comment:
What does this change do? You still have `unset(ARCHIVE_ENABLE_BZip2)` a few lines further down.